indico/core/db/sqlalchemy/custom/int_enum.py

```diff
@@ -64,15 +66,22 @@ def coerce_set_value(self, value):
             return None
         return self.enum(value)
 
-    def alembic_render_type(self, autogen_context):
+    def alembic_render_type(self, autogen_context, toplevel_code):
+        name = '_{}'.format(self.enum.__name__)
+        members = '\n'.join(['    {} = {}'.format(x.name, x.value) for x in self.enum])
```

Comment:
the `[]` isn't needed here; passing a generator to join() works fine

and I think `{!r}` would be slightly cleaner for the value, even though for an int it doesn't really matter whether you use the repr explicitly or not...